lib/ansible/playbook/play_context.py

```diff
@@ -242,11 +242,9 @@ class PlayContext(Base):
     _verbosity = FieldAttribute(isa='int', default=0)
     _only_tags = FieldAttribute(isa='set', default=set())
     _skip_tags = FieldAttribute(isa='set', default=set())
-    _check_mode = FieldAttribute(isa='bool', default=False)
```

Comment:
I assume this is being removed because it's redundant with the one in `lib/ansible/playbook/base.py`, but should the default be moved to that file?